### PR TITLE
Adding automated release

### PR DESCRIPTION
**Pull request by Adafede**
@cthoyt 

I guess this should do the trick. 

Btw, I was not able to test it on my fork because of the local changes...

### CODE REVIEW

**Comment by cthoyt** [comment]
The inaugural run of the action: https://github.com/biopragmatics/bioregistry/actions/runs/2378040486

**Comment by cthoyt** [comment]
Seems like everything worked alright except it named it "latest" instead of mirroring the tag name as the release title

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: marvinpinto/action-automatic-releases@v1.2.1
+        with:
+          repo_token: "${{ secrets.GITHUB_TOKEN }}"
+          automatic_release_tag: "latest"
```
Comment (cthoyt) [inline .github/workflows/release.yml:13]: can we change this bit to use the same name as the tag?

**Comment by Adafede** [comment]
Oh crap!

We should just remove  https://github.com/biopragmatics/bioregistry/blob/7d6f389d59a90ba4fa65a7a6870ffbda43b338a2/.github/workflows/release.yml#L13

competely and should be OK

File: .github/workflows/release.yml (hunk 1 above)
Comment (Adafede) [reply .github/workflows/release.yml:13]: Sorry as the PR was closed made a new one with the requested change

**Pull request merged**